tests/php/NetworkFunction/NetworkUseCaseTest.php

```diff
@@ -136,6 +138,48 @@ public function testMoreThanMaxPagesResultsInError() {
 		);
 	}
 
+	public function testDefaultEnableDisplayTitle() {
+		$request = $this->newBasicRequestModel();
+
+		$this->assertSame(
+			$request->defaultEnableDisplayTitle,
+			true
+		);
+		$this->assertSame(
+			$this->runAndReturnPresenter( $request )->getResponseModel()->enableDisplayTitle,
+			true
+		);
+	}
+
+	public function testOverrideEnableDisplayTitleFalse() {
+		$request = $this->newBasicRequestModel();
+		$request->functionArguments = [ 'enableDisplayTitle = true' ];
+
+		$this->assertSame(
+			$request->defaultEnableDisplayTitle,
+			true
+		);
+		$this->assertSame(
+			$this->runAndReturnPresenter( $request )->getResponseModel()->enableDisplayTitle,
+			true
+		);
+	}
+
+
+	public function testOverrideEnableDisplayTitleTrue() {
+		$request = $this->newBasicRequestModel();
+		$request->functionArguments = [ 'enableDisplayTitle = false' ];
+
+		$this->assertSame(
+			$request->defaultEnableDisplayTitle,
+			true
+		);
```

Comment:
Looks like each of those 3 tests only should have one assertion. First one about the default, and the other two about the response model.

Reply:
Agreed about the second two. Checking the request in the first one turns out not to be very useful, so I'm checking the response in that one as well.